ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/CliCommand.java

```diff
@@ -67,29 +92,106 @@ protected FileUtils getFileUtils() {
         return this.fileUtils;
     }
 
-    public abstract CliCommand init(final Arguments args) throws Exception;
+    /**
+     * Lookup help for '--help' COMMAND from resource file
+     * (Override or add help text to resources/com/spectralogic/dscli/help.properties
+     * @param command
+     * @return
+     */
+    public static String getLongHelp(final String command) {
+        return CommandHelpText.getHelpText(command.toUpperCase());
+    }
 
+    /**
+     * Convenience method to add args, parse, and get output-format
+     * @param requiredArgs List of options to be added as required use EMPTY_LIST for none
+     * @param optionalArgs List of options to be added as optional use EMPTY_LIST for none
+     * @param args
+     * @throws BadArgumentException
+     * @throws ParseException
+     */
+    protected void processCommandOptions(final List<Option> requiredArgs,
+                                         final List<Option> optionalArgs,
+                                         final Arguments args) throws BadArgumentException, ParseException {
+        addRequiredArguments(requiredArgs, args);
+        addOptionalArguments(optionalArgs, args);
+        args.parseCommandLine();
+        this.viewType = args.getOutputFormat();
+    }
 
-    // help for '--help' command
-    public String getLongHelp(final String command) {
-        return CommandHelpText.getHelpText(command) ;
+
+    /**
+     * Inits COMMAND to add all options, then prints usage description
+     * @param arguments Arguments object
+     */
+    public void printArgumentHelp(final Arguments arguments) {
+        try {
+            // init to install options
+            init(arguments);
+        } catch (final Exception e) {}
```

Comment:
should add a comment to make it clear that you are intentionally ignoring any thrown exception.